plugins/networklayer_tcp.c

```diff
@@ -426,8 +427,10 @@ ServerNetworkLayerTCP_getJobs(UA_ServerNetworkLayer *nl, UA_Job **jobs, UA_UInt1
     size_t j = 0;
     UA_ByteString buf = UA_BYTESTRING_NULL;
     for(size_t i = 0; i < layer->mappingsSize && j < (size_t)resultsize; i++) {
-        if(!UA_fd_isset(layer->mappings[i].sockfd, &fdset))
-            continue;
+        if(UA_fd_isset(layer->mappings[i].sockfd, &errset)) {}
```

Comment:
@jpfr I think the idea is, to call socket_recv on an 'errored' socket, to receive an error message and to free the socket starting from line 440